File: 0628-maximum-product-of-three-numbers/0628-maximum-product-of-three-numbers.cpp
class Solution {
public:
    int maximumProduct(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int n = nums.size();
        long long op1 = 1LL * nums[n-1] * nums[n-2] * nums[n-3];
        long long op2 = 1LL * nums[n-1] * nums[0] * nums[1];
        return max(op1, op2);
    }
};

File: 0628-maximum-product-of-three-numbers/0628-maximum-product-of-three-numbers.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maximumProduct = function(nums) {
    const n=nums.length;
    nums.sort((a,b)=>a-b);
    const ch1 = nums[n-1]*nums[n-2]*nums[n-3];
    const ch2 = nums[0]*nums[1]*nums[n-1];
    return Math.max(ch1,ch2);
};